src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -50,5 +51,18 @@ public class StorageConfiguration : ConfigurationCategory
         [SizeUnit(SizeUnit.Megabytes)]
         [ConfigurationEntry("Storage.MaxScratchBufferSizeInMb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public Size? MaxScratchBufferSize { get; set; }
+
+        [Description("Size of the batch that will be requested to the OS from disk when prefetching (value in powers of 2). Some OSs may not honor certain values. Experts only.")]
+        [DefaultValue(1024)]
+        [SizeUnit(SizeUnit.Kilobytes)]
+        [ConfigurationEntry("Storage.IOPrefetchBatchSize", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public Size IOPrefetchBatchSize { get; set; }
+
+        [Description("How many gigabytes of memory should be prefetched before restarting the prefetch tracker table. Experts only.")]
+        [DefaultValue(8)]
+        [SizeUnit(SizeUnit.Gigabytes)]
+        [ConfigurationEntry("Storage.IOPrefetchResetThreshold", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
Can you change `Storage.IOPrefetchResetThreshold` to `Storage.IOPrefetchResetThresholdInGb` ? The property name should stay as is, but the configuration entry needs to include the units.